http_logs/operations/default.json

```diff
@@ -79,6 +87,58 @@
         }
       }
     },
+    {
+      "name": "200s-in-range",
```

Comment:
These typify the kinds of queries that we hope are useful with runtime fields - ones where you can use the `@timestamp` to eliminate many options and then let runtime fields slowly check everything.